### PR TITLE
feature/78: Implemented sorting images by text

### PR DESCRIPTION
**Pull request by gusartus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "react-masonry-css": "1.0.12",
     "react-redux": "6.0.0",
     "react-router-dom": "4.3.1",
+    "react-string-replace": "^0.4.4",
```
Comment (alena-andr) [inline package.json]: remove ^

File: src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx [hunk 2]
```diff
@@ -154,30 +155,30 @@ export default class ImageCard extends Component<IProps, IState> {
 				</div>
 				<div className={styles.titles}>
 					<img className={styles.titles_img} src={ thumb } />
-					<div className={styles.titles_main}>{ distro }</div>
+					<div className={styles.titles_main}>{ distro.full_name_filtered }</div>
 					<div className={styles.titles_sub}>
-						<Trans i18nKey='Image.StartedAt' />: { moment(startedAt).fromNow() }
+						<Trans i18nKey='Image.StartedAt' />: { moment(started_at).fromNow() }
```
Comment (alena-andr) [inline src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx]: keep one style: remove extra space

File: src/modules/Dashboard/components/ImagesCardView/ImagesCardView.tsx [hunk 3]
```diff
@@ -26,17 +27,22 @@ const breakpointCols = {
...
-						key={index}
-						image={image}
-						onSelect={onSelect}
-						isActive={image.name === activeImage.name}
-					/>
-				))
-			}
-		</Masonry>
+		{
+			imagesByActiveStatus.length > 0
```
Comment (alena-andr) [inline src/modules/Dashboard/components/ImagesCardView/ImagesCardView.tsx]: remove "> 0"

File: package.json (hunk 1 above)
Comment (gusartus) [reply package.json]: Done

File: src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx (hunk 2 above)
Comment (gusartus) [reply src/modules/Dashboard/components/ImagesCardView/ImageCard/ImageCard.tsx]: Done

File: src/modules/Dashboard/components/ImagesCardView/ImagesCardView.tsx (hunk 3 above)
Comment (gusartus) [reply src/modules/Dashboard/components/ImagesCardView/ImagesCardView.tsx]: Done

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**